socli/socli.py

```diff
@@ -361,7 +362,51 @@ def main():
         search.google_search = False
         tag = namespace.tag
         has_tags()  # Adds tags to StackOverflow url (when not using google search.
-
+    if namespace.open_url:
+        import webbrowser
+        open_in_browser=False
+        display_condition=True
+        url_to_use=namespace.open_url[0]
+        if re.findall(r"^https:\/\/",url_to_use) !=[]:
+            pass
+        else:
+            url_to_use="https://" + url_to_use
+        try:
+            if url_to_use == "https://stackoverflow.com/questions/":
+                raise Exception('URL Error')
+            if url_to_use == "https://www.stackoverflow.com/questions/":
+                raise Exception('URL Error')
+            requests.get(url_to_use)
+        except requests.ConnectionError or Exception:
```

Comment:
Codacy found an issue: [Exception to catch is the result of a binary "or" operation](https://app.codacy.com/manual/gautamkrishnar/socli/pullRequest?prid=5814058)

Reply:
Um I don't think the try block is raising any ConnectionErrors anymore?

Reply:
@hedythedev ConnectionErrors is raised by `requests.get(url_to_use)`. It will still raise error if the connection fails. I will use a generic Exception here to get rid of this codacy warning,.